db/views/reporting_facility_states_v11.sql

```diff
@@ -0,0 +1,374 @@
+
+overdue_patients_aggregate.filtered_returned_after_call_with_result_remind_to_call_later,
+overdue_patients_aggregate.filtered_returned_after_call_with_result_remind_to_call_later * 100.0 / nullif(filtered_called_with_result_remind_to_call_later, 0) as filtered_percentage_returned_after_call_with_result_remind_to_call_later,
+
+overdue_patients_aggregate.filtered_returned_after_call_with_result_remove_from_overdue_list,
+overdue_patients_aggregate.filtered_returned_after_call_with_result_remove_from_overdue_list * 100.0 / nullif(filtered_called_with_result_remove_from_overdue_list, 0) as filtered_percentage_returned_after_call_with_result_remove_from_overdue_list
+
+FROM reporting_facilities rf
+INNER JOIN reporting_months cal
+-- ensure a row for every facility and month combination
+    ON TRUE
+LEFT OUTER JOIN registered_patients
+    ON registered_patients.month_date = cal.month_date
+    AND registered_patients.region_id = rf.facility_region_id
+LEFT OUTER JOIN registered_diabetes_patients
+    ON registered_diabetes_patients.month_date = cal.month_date
+    AND registered_diabetes_patients.region_id = rf.facility_region_id
+LEFT OUTER JOIN registered_hypertension_and_diabetes_patients
+    ON registered_hypertension_and_diabetes_patients.month_date = cal.month_date
+    AND registered_hypertension_and_diabetes_patients.region_id = rf.facility_region_id
+LEFT OUTER JOIN assigned_patients
+    ON assigned_patients.month_date = cal.month_date
+    AND assigned_patients.region_id = rf.facility_region_id
+LEFT OUTER JOIN assigned_diabetes_patients
+    ON assigned_diabetes_patients.month_date = cal.month_date
+    AND assigned_diabetes_patients.region_id = rf.facility_region_id
+LEFT OUTER JOIN adjusted_outcomes
+    ON adjusted_outcomes.month_date = cal.month_date
+    AND adjusted_outcomes.region_id = rf.facility_region_id
+LEFT OUTER JOIN adjusted_diabetes_outcomes
+    ON adjusted_diabetes_outcomes.month_date = cal.month_date
+    AND adjusted_diabetes_outcomes.region_id = rf.facility_region_id
+LEFT OUTER JOIN monthly_cohort_outcomes
+    ON monthly_cohort_outcomes.month_date = cal.month_date
+    AND monthly_cohort_outcomes.region_id = rf.facility_region_id
+LEFT OUTER JOIN monthly_overdue_calls
+    ON monthly_overdue_calls.month_date = cal.month_date
+    AND monthly_overdue_calls.region_id = rf.facility_region_id
+LEFT OUTER JOIN monthly_follow_ups
+    ON monthly_follow_ups.month_date = cal.month_date
+    AND monthly_follow_ups.facility_id = rf.facility_id
+LEFT OUTER JOIN monthly_diabetes_follow_ups
+    ON monthly_diabetes_follow_ups.month_date = cal.month_date
+    AND monthly_diabetes_follow_ups.facility_id = rf.facility_id
+LEFT OUTER JOIN reporting_facility_appointment_scheduled_days
+    ON reporting_facility_appointment_scheduled_days.month_date = cal.month_date
+    AND reporting_facility_appointment_scheduled_days.facility_id = rf.facility_id
+LEFT OUTER JOIN overdue_patients_aggregate
```

Comment:
Can we not call this `monthly_overdue_patients`?

Reply:
updated to `monthly_overdue_patients`